helpers.js

```diff
@@ -0,0 +1,55 @@
+const log4js  = require('log4js');
+const LOG     = log4js.getLogger('helpers');
+const format  = require('format');
+const _       = require('lodash');
+
+// Using getMessages will automatically use already cached messages when available,
+//     and will fetch more messages only if needed. This way we don't have to rely
+//     on the promise function structure every time we want to retrieve a message
+exports.getMessages = async function getMessages(message) {
+
+  // Current messages in the Discord client's cache
+  let cachedMessages = message.channel.messages.array()[0].channel.messages;
+  
+  // If there are only a couple of items in the cache...
+  if (cachedMessages.array().length < 10) {
+    LOG.info('Message cache empty for this channel; fetching messages.');
+    // Use fetchMessages() to get messages from before we started caching
+    cachedMessages = await message.channel.fetchMessages({limit: 100});
+  }
+
+  return cachedMessages;
+}
+
+
+// Randomly look through all results until we get a message that 
+//     isnt bot generated or a bot command
+exports.filterMessages = function filterMessage(messages) {
```

Comment:
why don't we call this like `getUserMessage` or `getRandomNonBotMessage` haha? `filterMessages is a little ambiguous.